Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -766,6 +766,51 @@ public void test_checkWrongCardsDataInSets() {
         }
     }
 
+    @Test
+    @Ignore
+    public void test_checkWrongFullArtAndRetro() {
+        Collection<String> errorsList = new ArrayList<>();
+        Collection<ExpansionSet> xmageSets = Sets.getInstance().values();
+
+        // CHECK: wrong card numbers
+        for (ExpansionSet set : xmageSets) {
+            if (skipListHaveName(SKIP_LIST_WRONG_CARD_NUMBERS, set.getCode())) {
+                continue;
+            }
+
+            for (ExpansionSet.SetCardInfo card : set.getSetCardInfo()) {
+                MtgJsonCard jsonCard = MtgJsonService.cardFromSet(set.getCode(), card.getName(), card.getCardNumber());
+                if (jsonCard == null) {
+                    continue;
+                }
+
+                // CHECK: poster promoType and/or textless must use full art setting
+                if (((jsonCard.promoTypes != null && jsonCard.promoTypes.contains("poster")) || jsonCard.isTextless) && !card.isFullArt()) {
+                    errorsList.add("Error: card must use full art setting: "
+                            + set.getCode() + " - " + set.getName() + " - " + card.getName() + " - " + card.getCardNumber());
+                }
+
+               // CHECK: full art lands must use full art setting
+               boolean isLand = card.getRarity().equals(Rarity.LAND);
+               if (jsonCard.isFullArt && isLand && !card.isFullArt()) {
+                   errorsList.add("Error: card must use full art setting: "
+                           + set.getCode() + " - " + set.getName() + " - " + card.getName() + " - " + card.getCardNumber());
+               }
+
+                // CHECK: must use retro frame setting
+                if ((jsonCard.frameVersion.equals("1993") || jsonCard.frameVersion.equals("1997")) && !card.isRetroFrame()) {
+                    errorsList.add("Error: card must use retro art setting: "
+                            + set.getCode() + " - " + set.getName() + " - " + card.getName() + " - " + card.getCardNumber());
+                }
+            }
```

Comment:
All other cards can't use full art settings and must raise error at the end? E.g. if card.isRetroFrame() then add error about wrong full art setting like old code does:

<img width="1359" height="174" alt="shot_250830_042841" src="https://github.com/user-attachments/assets/123e2747-f049-4ed3-9dfe-85b59e5b871f" />